.github/workflows/docker-hub.yml

```diff
@@ -0,0 +1,47 @@
+# Ref: https://github.com/docker/build-push-action#usage
+name: Build Docker container
+
+on:
+  push:
+    branches:
+      - development
+      - feature/gh-action-build-docker
```

Comment:
```suggestion
```
I believe, once this this is merged, we should only watch for pushes on `development`, shouldn't we?

Reply:
Right